diff --git a/voxygen/src/audio/soundcache.rs b/voxygen/src/audio/soundcache.rs
index 8982c7a232..684ea1950f 100644
--- a/voxygen/src/audio/soundcache.rs
+++ b/voxygen/src/audio/soundcache.rs
@@ -174,7 +174,7 @@ impl Loader<StreamedOggSound> for StreamedSoundLoader {
 impl OggSound {
     pub fn empty() -> OggSound {
         SoundLoader::load(
-            Cow::Borrowed(include_bytes!("../../../assets/voxygen/audio/null.ogg")),
+            Cow::Borrowed(include_bytes!(env!("VOXYGEN_NULL_SOUND_PATH"))),
             "ogg",
         )
         .unwrap()
